Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -107,6 +106,40 @@ void AsyncScrollingCoordinator::handleWheelEventPhase(ScrollingNodeID nodeID, Pl
 }
 #endif
 
+ScrollingStateTree& AsyncScrollingCoordinator::ensureScrollingStateTreeForRootFrameID(FrameIdentifier identifier)
```

Comment:
This needs to assert that `identifier` is a root frame identifier. Please also rename the parameter.

Reply:
There is currently no way to assert that it is a root frame identifier, but we could easily add such a way by keeping extra bookkeeping but only if ASSERT_ENABLED is true.

Reply:
Fixed.